doc/source/development/contributing.rst

```diff
@@ -569,8 +569,8 @@ do not make sudden changes to the code that could have the potential to break
 a lot of user code as a result, that is, we need it to be as *backwards compatible*
 as possible to avoid mass breakages.
 
-Additional standards are outlined on the `code style wiki
-page <https://github.com/pandas-dev/pandas/wiki/Code-Style-and-Conventions>`_.
+Additional standards are outlined on the (Placeholder as a reminder to ask the
+devs for the help on how to properly link the new created file)
```

Comment:
I need help linking the new file here.

Reply:
Don't remember the exact syntax, but you have plenty of examples of links to labels in the docs.